editoast/src/core/mq_client.rs

```diff
@@ -51,6 +49,12 @@ pub enum Error {
     #[error("Connection does not exist")]
     #[editoast_error(status = "500")]
     ConnectionDoesNotExist,
+    #[error("Cannot create the pool")]
+    #[editoast_error(status = "500")]
+    CeatePoolLapin(CreatePoolError),
```

Comment:
There are other occurrences, please `ctrl+F`:
```suggestion
    CreatePoolLapin(CreatePoolError),
```